apps/admin/frontend/src/screens/write_ins_adjudication_screen.tsx

```diff
@@ -87,6 +92,11 @@ const ContestTitle = styled(H2)`
   margin: 0 !important;
 `;
 
+const ContestSubTitle = styled.div`
+  padding-top: 0.5rem;
+  word-break: break-word;
+`;
```

Comment:
I wonder if this should be styled to pop more, since we want the user to focus on it when viewing a BMD write-in.

Reply:
adding primary color styling after experimenting